src/applications/simple-forms/20-10207/sass/10207-pp.scss

```diff
@@ -5,3 +5,13 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../../platform/forms/sass/m-form-confirmation";
+
+legend:first-child > h3 {
```

Comment:
what's this for?

is this for titleUI? do we need different styling?

Reply:
Not sure.  I'll check, and ping you if a titleUI refactor's warranted.